Update configure-spring-boot-starter-java-app-with-azure-active-direc… (#547)
MicrosoftDocs/azure-dev-docs: articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-active-directory.md

1. When prompted, download the project to a path on your local computer.

> [!NOTE]

> > We have released Spring Boot Starter for Azure Active Directory [3.6.1](https://github.com/Azure/azure-sdk-for-java/blob/hotfix/azure-spring-boot_3.6.1/sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md) to address the following CVE report [CVE-2021-22119: Denial-of-Service attack with spring-security-oauth2-client](https://tanzu.vmware.com/security/cve-2021-22119). If you are using the older version, plesae upgrade it to 3.6.1 and above. 

Reviewer: ```suggestion
> We've released Spring Boot Starter for Azure Active Directory [3.6.1](https://github.com/Azure/azure-sdk-for-java/blob/hotfix/azure-spring-boot_3.6.1/sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md) to address the following CVE report [CVE-2021-22119: Denial-of-Service attack with spring-security-oauth2-client](https://tanzu.vmware.com/security/cve-2021-22119). If you're using the older version, please upgrade it to 3.6.1 or above. 
```
